Formula/ansilove.rb

```diff
@@ -0,0 +1,19 @@
+class Ansilove < Formula
+  desc "ANSi / ASCII art to PNG converter in C"
+  homepage "https://www.ansilove.org/"
+  url "https://github.com/ansilove/ansilove/archive/3.0.8.tar.gz"
+  sha256 "655472d31a9ec5a5ba29f5c15485d8647526ff8b0c9968d71f4267a9aa9bb544"
+
+  depends_on "cmake" => :build
+  depends_on "gd"
+
+  def install
+    system "cmake", "."
+    system "make"
+    bin.install "ansilove"
+  end
+
+  test do
+    system "#{bin}/ansilove", "-v"
```

Comment:
We need a functional test, that does something and check the output.

Reply:
What is a functional test for a utility like this?  Should I wget an ansi file from somewhere and try to convert it to png (that's what this utility does).

Reply:
Can you install one of the example ASCII files, convert it to PNG and check that the PNG file has indeed been created?